lib/transform.js

```diff
@@ -8,8 +8,8 @@ var through          =  require('through')
 function requireDependencies(src) {
   var deps = findDependencies(src);
   if (!deps.length) return '';
-  return '/* proxyquireify injected requires to make browserify include dependencies in the bundle */;' +
-    requireDeps(deps);
+  return '/* proxyquireify injected requires to make browserify include dependencies in the bundle */ /* istanbul ignore next */; (function __makeBrowserifyIncludeModule__() { ' +
+    requireDeps(deps) + '});\n\n';
```

Comment:
Please remove `\n\n` since we don't want to change the user's code too much. 
Injecting a line right in front of the other was done to preserve line numbers between code that runs with and without proxyquireify.